config.py

```diff
@@ -1831,14 +1856,30 @@ def get_datatype() -> Literal['meg', 'eeg']:
 
 
 def _get_reject(
-    reject: Optional[Dict[str, float]],
-    ch_types: Iterable[Literal['meg', 'mag', 'grad', 'eeg']]
+    *,
+    subject: Optional[str] = None,
+    session: Optional[str] = None,
+    reject: Optional[Union[Dict[str, float], Literal['autoreject_global']]],
+    ch_types: Iterable[Literal['meg', 'mag', 'grad', 'eeg']],
+    epochs: Optional[mne.BaseEpochs] = None
 ) -> Dict[str, float]:
     if reject is None:
         return dict()
 
-    reject = reject.copy()
+    if reject == 'autoreject_global':
+        # Automated threshold calculation requested
+        import autoreject
 
+        msg = 'Generating rejection thresholds using autoreject …'
+        logger.info(gen_log_message(message=msg, subject=subject,
+                                    session=session))
+        reject = autoreject.get_rejection_threshold(
+            epochs=epochs, ch_types=ch_types, decim=decim, verbose=False
```

Comment:
@agramfort 
I'm passing `ch_types` here because without doing it, I'll lose >50% of my epochs on ERP CORE thanks to the EOG threshold that `autoreject` generates otherwise